src/magic.cpp

```diff
@@ -317,6 +317,8 @@ void spell_type::load( const JsonObject &jo, const std::string & )
     optional( jo, was_loaded, "base_energy_cost", base_energy_cost, 0 );
     optional( jo, was_loaded, "final_energy_cost", final_energy_cost, base_energy_cost );
     optional( jo, was_loaded, "energy_increment", energy_increment, 0.0f );
+    optional( jo, was_loaded, "base_arm_encumbrance_value", base_arm_encumbrance_value, 20 );
```

Comment:
```suggestion
    optional( jo, was_loaded, "arm_encum_thresh", arm_encumbrance_threshold, 20 );
```
Perhaps this would be a better set of names? And similar can go for the leg values too